docs/source/onnx.rst

```diff
@@ -18,7 +18,7 @@ saves the resulting traced model to ``alexnet.proto``::
 
     # providing these is optional, but makes working with the
     # converted model nicer.
```

Comment:
ideally on this line could you say "parameters come after real inputs, and providing names for them is optional"?